### DISCUSSION TITLE
Use the tracing design in Presto

### DISCUSSION THREAD

**Discussion by jerryleooo** [trinodb/trino#11158] [Ideas]
Currently, there have been some efforts/proposals on a better tracing:

https://github.com/trinodb/trino/issues/1826
https://github.com/trinodb/trino/pull/2010
https://github.com/trinodb/trino/issues/4207
https://github.com/trinodb/trino/pull/9924

But seems no conclusion for now.

I found PrestoDB has relative works: https://github.com/prestodb/presto/tree/6af495cb192279ae6de56bbce50c26d6f217d26b/presto-main/src/main/java/com/facebook/presto/tracing

Then use `EventListener` to use the tracer.

Not sure if we will consider using the same solution?

**Comment by jerryleooo** [comment]
https://github.com/trinodb/trino/pull/11244

**Comment by KingMCMS** [comment]
Am Boden Zerstört 

https://open.spotify.com/album/4c4jwrdrK915N7krhQCVLo?si=8BIfQvtcRn6dHz5ZQYbSBA

**Reply by jerryleooo** [reply]
?
